activerecord/test/cases/migration/exclusion_constraint_test.rb

```diff
@@ -74,6 +149,29 @@ def test_added_exclusion_constraint_ensures_valid_values
           end
         end
 
+        def test_added_deferrable_initially_immediate_exclusion_constraint
```

Comment:
This test gets an error if it runs isolated.

```ruby
$ ARCONN=postgresql bin/test test/cases/migration/exclusion_constraint_test.rb -n test_added_deferrable_initially_immediate_exclusion_constraint
Using postgresql
Run options: -n test_added_deferrable_initially_immediate_exclusion_constraint --seed 54897

# Running:

E

Error:
ActiveRecord::Migration::ExclusionConstraintTest#test_added_deferrable_initially_immediate_exclusion_constraint:
NameError: uninitialized constant ActiveRecord::Migration::ExclusionConstraintTest::Section
    /home/yahonda/src/github.com/rails/rails/activerecord/test/cases/migration/exclusion_constraint_test.rb:164:in `block in test_added_deferrable_initially_immediate_exclusion_constraint'
    /home/yahonda/src/github.com/rails/rails/activesupport/lib/active_support/testing/assertions.rb:48:in `assert_nothing_raised'
    /home/yahonda/src/github.com/rails/rails/activerecord/test/cases/migration/exclusion_constraint_test.rb:163:in `test_added_deferrable_initially_immediate_exclusion_constraint'


bin/test test/cases/migration/exclusion_constraint_test.rb:152



Finished in 0.025691s, 38.9244 runs/s, 38.9244 assertions/s.
1 runs, 1 assertions, 0 failures, 1 errors, 0 skips
$
```

Reply:
Fixed to use `Invoice` instead of `Section`.